### PR TITLE
Adding documentation about type inferring

### PR DESCRIPTION
**Pull request by yordis**
Related to #610 #611

### CODE REVIEW

**Comment by yordis** [comment]
@mweststrate Do you think this could make it into `master`?

**Comment by mweststrate** [comment]
@yordis I was about to say yes, but recently I saw somebody using an even nicer pattern. Wanted to investigate that. Saddly I didn't copy the link and now I cant find it anymore. :'(

Somewhere next week I want to batch process a lot of MST PR's and create a new release. If I didn't find something else I'll merge this one!

File: README.md [hunk 1]
```diff
@@ -1215,6 +1217,24 @@ const Example = types
         return twiceUpperProp();
       },
     };
+  })
+  .actions(self => {
+    // Don't forget that async operations HAVE
+    // to use `flow( ... )`.
+    const fetchData = flow(function *() {
```
[the remark this answers is not in the document]
Comment (FredyC) [reply README.md]: @yordis So you liked my comment and yet you haven't changed it? :)

**Comment by mweststrate** [comment]
Merged, thanks!

**Pull request merged**